tests/Integration/tests/index/expected/index.html

```diff
@@ -0,0 +1,168 @@
+<!DOCTYPE html>
```

Comment:
Do we really need the whole HTML structure or just a part of it? So checking in the texts if the expected output is part of the whole actual HTML structure? This makes maintaining more easy, when we change the head output, for example.

Reply:
We treat the output like a baseline. You are alerted about any change and if the change is fine you can just copy the accepted output back into the integration. In the guides this strategy worked fine for us. Each Commit which causes a change in the output will then also be in the Git-History of that file.

Reply:
Okay, may be error-prone, if you have many changes in the output and some erroneous one slipps through.